extra_data/components.py

```diff
@@ -1109,7 +1256,7 @@ def identify_multimod_detectors(
     *clses* may be a list of acceptable detector classes to check.
     """
     if clses is None:
-        clses = [AGIPD1M, DSSC1M, LPD1M]
+        clses = detectors.list
```

Comment:
This too is technically a breaking change - previously if you used this function on data containing AGIPD & JUNGFRAU detectors, it would find the AGIPD detector, now it can error (if `single=True`). But I'm not too concerned, as this function is probably mostly used by us, building generic tools, rather than scientists who will normally write code for a specific detector.

We might want to adjust the docstring talking about "Various detectors record data in a similar format..." though, because JUNGFRAU is not as similar as AGIPD/LPD/DSSC.

Reply:
Since we are now using this function to identify multimod detector class in 'cli/make_virtual_cxi.py' this change is required. I've tried to adjust the docstring in https://github.com/European-XFEL/EXtra-data/commit/156b8bf353b59b107950a2baf566352c061acc4c - please let me know your opinion or suggestions.